docs/governance.rst

```diff
@@ -0,0 +1,244 @@
+2. Every modification (including a correction of a typo,
+   adding a new Contributor, an extension,
+   or others) or proposal to release a new version needs to be done via a
+   Pull Request (PR) to the Repository.
+3. Anyone can open an Issue or a PR (this action is not limited to Contributors).
+4. A PR is eligible to be merged if and only if these conditions are met:
+
+   a) The PR features at least two `Reviews that Approve <https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews>`_
+      the PR of which neither is the author of the PR. 
+      The reviews should be made after the last commit in the PR (equivalent to 
+      `Stale review dismissal <https://help.github.com/articles/enabling-required-reviews-for-pull-requests/>`_
+      option on GitHub). If a second review requests minor changes after
+      another reviewer approved the PR, the first review does not need
+      to re-review.
+   b) Does not feature any `Reviews that Request changes <https://help.github.com/articles/about-required-reviews-for-pull-requests/>`_.
+      That is, if someone asked for changes, the PR should not be merged just
+      because two other people approve it.
+   c) Is not a Draft PR. That is the PR author says it is ready for review.
+   d) Passes all automated tests.
+   e) Is not proposing a new release.
+   f) The steering committee has not added extra restrictions. For example, if
+      a PR is a non-trival change, the steering committee can create a system
+      to get feedback from more than just two reviewers before merging.
+5. After consultation with contributors, the steering committee can decide
+   to merge any PR - even if it's not eligible to merge according to Rule 4.
+6. Anyone can Review a PR and request changes. If a community
+   member requests changes they need to provide an explanation regarding what
+   changes should be made and justification of their importance. Reviews
+   requesting changes can also be used to request more time to review a PR.
+7. A reviewer who requested changes can dismiss their own review, if
+   they decide their requested changes are no longer necessary, or approve
+   changes that address the issue underlying their change request.
+8. If the author of a PR and a reviewer who requests changes cannot find a
+   solution that would lead to: (1) The author closing the PR without merging
+   (2) The reviewer accepting requested changes or (3) The dismissing their
+   review, so that the PR can be approved and merged, then the disagreement
+   will be resolved with a vote.
+9. Rules governing voting:
+
+   a) A vote can be triggered by any Maintainer, but only after 5 working days
+      from the time a Review Requesting Changes is made. If a PR has a disagreement
+      that required a vote, there must be at least 15 days from the conclusion
+      of the first vote before another vote can be triggered.
+   b) Only Maintainers can vote and each Maintainer gets one vote.
```

Comment:
I'd prefer if this was Contributors rather than Maintainers.

Is there a reason to go with Maintainers here, @tsalo @handwerkerd ?

Reply:
I was keeping this as it was. The advantage of Maintainers is a that conflict is resolved by a clear number of people who are invested in the greater whole. The disadvantage is that there's no guarantee that Maintainers will understand a specific issue more than other contributors. As long as votes won't get slowed down waiting for a quorum, I'm find changing to Contributors.

Reply:
I am also fine with changing to Contributors, particularly since it could be a vote on user-facing behavior.

Reply:
```suggestion
   b) Only Contributors can vote and each Contributor gets one vote.
```